sktime/forecasting/exp_smoothing.py

```diff
@@ -97,6 +147,16 @@ def __init__(
         self.initial_trend = initial_trend
         self.initial_seasonal = initial_seasonal
         self.initialization_method = initialization_method
+        self.smoothing_level = smoothing_level
+        self.smoothing_trend = smoothing_trend
+        self.smoothing_seasonal = smoothing_seasonal
+        self.damping_trend = damping_trend
+        self.optimized = optimized
+        self.remove_bias = remove_bias
+        self.start_params = start_params
+        self.method = method
+        self.minimize_kwargs = minimize_kwargs
+        self.use_brute = use_brute
 
         super(ExponentialSmoothing, self).__init__()
```

Comment:
is this call necessary when no argument are being passed?

Reply:
they are being passed later, in `fit`? Or what do you mean?

Reply:
Sorry, I'm only referring to the call to `super`

Reply:
> is this call necessary when no argument are being passed?

ah, I see - yes, since it handles setting of the `_is_fitted` flag and, potentially, related concerns.

Reply:
I can only see `_StatsModelAdapter` setting `self._forecaster = None` and  `self._fitted_forecaster = None`, shouldn't the `_is_fitted` be set in `fit`?

Reply:
all fittable estimators should set `_is_fitted` to `False` in `__init__`.
In this case, it should happen two levels higher (after two `super` calls), in `BaseEstimator`.

Reply:
I think there is something to be said about the growing inheritance depth that does not make it straightforward to either review or write code extending functionality. That is probably a broader design discussion, but I want to make a comment here since it a good illustration of that point.